pages/implementation/reference/javascript.mdx

```diff
@@ -37,26 +37,19 @@ You can also <Link passHref href="/explore-tracking-plan/download-or-copy-avo-fi
 
 Import Avo from the generated file and initialize it by calling the `initAvo` method before tracking
 
-```javascript
-function initAvo(options, systemProperties, 
-                 destinationOptions, [customDestDestination])
-```
-
-The actual parameters depend on your tracking plan setup, see the parameters explanation in <Link passHref href="/implementation/devs-101#avo-generated-file-init"><a>the initialization overview</a></Link>
-
-and you call it like this
-
 ```javascript
 import Avo from './Avo';
 
-Avo.initAvo({ env: 'dev' }, { ...systemProperties }, {});
+Avo.initAvo({ env: 'dev' }, [systemProperties], [destinationOptions], [customDestDestination]);
 ```
 
+The actual parameters depend on your tracking plan setup, see the parameters explanation in <Link passHref href="/implementation/reference/javascript#refeernce#initAvo#arguments"><a>the reference below</a></Link>
```

Comment:
Linking this link in the Vercel build doesn't take me to the Reference section 🤔

Reply:
```suggestion
The actual parameters depend on your tracking plan setup, see the parameters explanation in <Link passHref href="/implementation/reference/javascript#refeernce#initAvo#arguments"><a>the reference below</a></Link>.
```